MergerService/Utils/TaskUtils.cs

```diff
@@ -1,8 +1,20 @@
+using System.Diagnostics;
+
 namespace MergerService.Utils
 {
     // TODO: rename all utils to clients
     public class TaskUtils
     {
+        private ILogger _logger;
+        private ActivitySource _activitySource;
+
+        public TaskUtils(ILogger<TaskUtils> logger, ActivitySource activitySource)
+        {
+            this._logger = logger;
+            this._activitySource = activitySource;
+            //TODO: add tracing and logging
```

Comment:
?

Reply:
this class isn't written yet so there is no place to add the logging yet.
it is remainder for when this class is implemented properly